Nothing found at https://oaitriton.blob.core.windows.net/public/llvm-builds/llvm-61f8a7f6-centos-x64.tar.gz

[TL;DR]
There should be an llvm pre-build in https://oaitriton.blob.core.windows.net/public/llvm-builds/llvm-61f8a7f6-centos-x64.tar.gz, but there is not.

[061707a47105dda6fcf6a1319db6bf17870cd334](https://github.com/intel/intel-xpu-backend-for-triton/pull/2470) is used in pytorch upstream triton commit pin.
https://github.com/pytorch/pytorch/pull/137886 is the relevant PR.

# Additional Context

```bash
Note: switching to '061707a47105dda6fcf6a1319db6bf17870cd334'.
You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

git switch -c <new-branch-name>

HEAD is now at 061707a4 Update version to 3.1.0 (#2470)
Applying patch_setup_py() for llvm-build package failed: Can't find $https://tritonlang.blob.core.windows.net/llvm-builds/ in the input. If you are trying to build a newer version of Triton, you can ignore this.
...
/opt/python/cp310-cp310/lib/python3.10/site-packages/setuptools/dist.py:561: UserWarning: The version specified ('3.1.0+git061707a4+061707a471') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
warnings.warn(
running bdist_wheel
running build
running build_py
running build_ext
downloading and extracting https://oaitriton.blob.core.windows.net/public/llvm-builds/llvm-61f8a7f6-centos-x64.tar.gz ...
error: HTTP Error 404: The specified blob does not exist.
Traceback (most recent call last):
File "/pytorch/.github/scripts/build_triton_wheel.py", line 223, in <module>
   main()
File "/pytorch/.github/scripts/build_triton_wheel.py", line 210, in main
   build_triton(
 File "/pytorch/.github/scripts/build_triton_wheel.py", line 180, in build_triton
   check_call(
File "/opt/python/cp310-cp310/lib/python3.10/subprocess.py", line 369, in check_call
  raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/opt/python/cp310-cp310/bin/python', 'setup.py', 'bdist_wheel']' returned non-zero exit status 1.
##[error]Process completed with exit code 1.
```

--- a/python/setup.py
+++ b/python/setup.py
@@ -207,7 +207,7 @@ def get_llvm_package_info():
     with open(llvm_hash_path, "r") as llvm_hash_file:
         rev = llvm_hash_file.read(8)
     name = f"llvm-{rev}-{system_suffix}"
-    url = f"https://oaitriton.blob.core.windows.net/public/llvm-builds/{name}.tar.gz"
+    url = f"https://github.com/intel/intel-xpu-backend-for-triton/releases/download/llvm-{rev}/{name}.tar.gz"
     return Package("llvm", name, url, "LLVM_INCLUDE_DIRS", "LLVM_LIBRARY_DIR", "LLVM_SYSPATH")
 
 

